### PR TITLE
 added sign up page and its styles

### PR DESCRIPTION
**Pull request by poojakelkar**
sign up
- creating signup page for quiz app

### CODE REVIEW

File: src/pages/Signup/index.jsx [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+
 const Signup = () => {
-  return <div>Sign Up</div>;
+  return (
+    <div className="loginUser">
+      <div className="topSide">
+        <div className="logoImg">
+          <img
+            classname="logo"
+            src="https://cdn-icons-png.flaticon.com/512/2784/2784530.png"
```
Comment (chanchal-panpaliya) [inline src/pages/Signup/index.jsx]: replace -> import img_name from '../../imgpath' ;

File: src/pages/Signup/styles.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    border: none;
+    cursor: pointer;
+    color: white;
+    font-weight: 500;
+    font-size: 18px;
+    background-color: var(--button-color);
+}
+span {
+    cursor: pointer;
+    font-size: 18px;
```
Comment (chanchal-panpaliya) [inline src/pages/Signup/styles.css]: changed units - rem

File: src/pages/Signup/styles.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+.link {
+    text-decoration: none;
+    color: white;
+    padding: 0rem;
```
Comment (chanchal-panpaliya) [inline src/pages/Signup/styles.css]: if padding:0 , remove padding.

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good . need refactor .

**Review by shrushti2000 — APPROVED** [review]

**Pull request merged**